### PR TITLE
ref: publish with craft

### PR DESCRIPTION
**Pull request by bruno-garcia**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -23,3 +23,15 @@ after_success:
...
+    urls:
+      - https://zeus.ci/hooks/b6bbaf86-c598-11e8-aa19-0a580a280112/public/provider/travis/webhook
+    on_success: always
+    on_failure: always
+    on_start: always
+    on_cancel: always
+    on_error: always
+after_script:
+  - npm install -g @zeus-ci/cli
+  - zeus upload -t "mime/type" path/to/artifact
```
Comment (tonyo) [inline .travis.yml]: :)

File: .appveyor.yml [hunk 2]
```diff
@@ -4,6 +4,8 @@ environment:
   global:
     DOTNET_SKIP_FIRST_TIME_EXPERIENCE: 1
     DOTNET_CLI_TELEMETRY_OPTOUT: 1
+  ZEUS_HOOK_BASE:
+    secure: dcqtt6sxxBV0tDkXmPZOy/mS9w6lpsiR4cz0HHfryT5/NHNfboPH5/VaOFvOXnh2gD3ftUutQ6h0GDUWT9HrKhZoqq2kqv20pevaRU0MOBTCJ92HiW6vSSkT2niMA/ZvWFtQGO2tdPURILtZ99aeZ7sirOjrSwRRoy/bZpo6Uzg=
```
Comment (tonyo) [inline .appveyor.yml]: Hmm, looks like ZEUS_HOOK_BASE was not added to the repo settings in Appveyor

File: .travis.yml (hunk 1 above)
Comment (bruno-garcia) [reply .travis.yml]: :)

File: .appveyor.yml (hunk 2 above)
Comment (bruno-garcia) [reply .appveyor.yml]: I added there and then realized there's no need to add in both. It's either added via the UI or via the config with `secure`. 
I've opted to add it here only then.

**Pull request merged**